schema/article.js

```diff
@@ -24,6 +25,10 @@ let ArticleType = new GraphQLObjectType({
     author: {
       type: AuthorType,
       resolve: ({ author }) => author
+    },
+    thumbnail_image: {
+      type: Image.type,
+      resolve: ({ thumbnail_image }) => thumbnail_image
```

Comment:
My thinking is: if this is a place where we're gluing together disparate APIs then it makes some sense to try to normalize them. It *looks* to me like that should be a nested object. But I'm rarely a consumer of Positron, so it's hard for me to say